demobot.py

```diff
@@ -45,22 +45,5 @@ async def on_ready():
     """
     print(f'{bot.user.name} has connected to Discord!')
 
-
-@bot.event
-async def on_command_error(ctx, error):
-    """
-    Handle the Error message in a nice way.
-    """
-    if isinstance(error, commands.errors.CheckFailure):
-        await ctx.send('You do not have the correct role for this command.')
-    elif isinstance(error, commands.errors.MissingRequiredArgument):
-        await ctx.send('You are missing a required argument.')
-    elif isinstance(error, commands.errors.CommandNotFound):
-        await ctx.send('Sorry, I didn\'t get that.')
```

Comment:
Only remove this line not all the error checking. Leave the case in though so it doesn't do the default case for any other error.

Reply:
problem is now the elif doesn't do anything and the elif fails

Reply:
Oh ye python can't have it empty, so try add a `pass` into it.